registry/storage/driver/middleware/redirect/middleware.go

```diff
@@ -0,0 +1,47 @@
+package middleware
+
+import (
+	"fmt"
```

Comment:
Can you separate the go/non-go imports and reorder these please? (gofmt should do this)